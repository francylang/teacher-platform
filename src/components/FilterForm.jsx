import React, { Component } from 'react';
import { math6RP, math6NS, math6EE, math6G, math6SP, math7RP, math7NS, math7EE, math7G, math7SP, math8NS, math8EE, math8F, math8G, math8SP, mathSix, mathSeven, mathEight } from '../utils/standards';

class FilterForm extends Component {
  constructor() {
    super();
    this.state = {
      selected: [],
      showingMath6: true,
      showingMath7: false,
      showingMath8: false,
    };
  };

  showOptions(standardsArray, state) {
    if (state) {
      return standardsArray.map(standard => {
        let firstChar = standard.charAt(0);
        if (firstChar === '6' || firstChar === '7' || firstChar === '8') {
          return (
            <div className="standard-checkbox" id={firstChar} >
              <input type="checkbox" id={standard} name={standard} value={standard}></input>
              <label htmlFor={standard}>{standard}</label>
            </div>
          );
        } else {
          return (
            <h4>{standard}</h4>
          );
        }
      });
    }
  };

  showMath6Tags() {
    this.setState({ showingMath6: true, showingMath7: false, showingMath8: false });
  };

  showMath7Tags() {
    console.log('show math 7');
    this.setState({ showingMath6: false, showingMath7: true, showingMath8: false });
  };

  showMath8Tags() {
    console.log('show math 8');
    this.setState({ showingMath6: false, showingMath7: false, showingMath8: true });
  };

  render() {
    return (
      <section>
        <h3>Click on a grade level tab to choose standard(s):</h3>
        <button onClick={() => this.showMath6Tags()}>
          Grade 6
        </button>
        <button onClick={() => this.showMath7Tags()}>
          Grade 7
        </button>
        <button onClick={() => this.showMath8Tags()}>
          Grade 8
        </button>
        <section className="options-container">
          <article className="options">
            { this.showOptions(mathSix, this.state.showingMath6) }
          </article>
          <article className="options">
            { this.showOptions(mathSeven, this.state.showingMath7) }
          </article>
          <article className="options">
            { this.showOptions(mathEight, this.state.showingMath8) }
          </article>
        </section>

        <button>FIND discussions based on what I checked!!</button>
      </section>
    );
  }
}

export default FilterForm;
