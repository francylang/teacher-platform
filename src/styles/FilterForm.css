#R, #S, #T, #G, #E, #F {
  outline: 2px solid yellow; }

.options-container {
  outline: 1px solid blue; }

.options {
  display: flex;
  flex-direction: row;
  align-items: flex-start; }

.standard-checkbox {
  margin: 4px; }

.icon {
  background-image: url(../assets/check.svg);
  height: 5px;
  width: 5px;
  background-repeat: no-repeat;
  background-size: cover;
  border: none; }
