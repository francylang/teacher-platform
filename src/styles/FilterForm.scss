#R, #S, #T, #G, #E, #F {
  outline: 2px solid yellow;
}

.filter-section {
  // width: 30%;
}

.options-container {
  outline: 1px solid blue;
}

.options {
  display: flex;
  flex-direction: row;
  align-items: flex-start;
}

.standard-checkbox {
  // width: 40%;
}
