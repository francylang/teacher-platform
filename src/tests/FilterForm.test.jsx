import React from 'react';
import ReactDOM from 'react-dom';
import FilterForm from '../components/FilterForm';
import { shallow } from 'enzyme';
import testSetup from './testSetup';

describe('<FilterForm />', () => {
  let wrapper;
  let mockFn = jest.fn();

  beforeEach(() => {
    wrapper = shallow(<FilterForm />);
  });

  it('should exist', () => {
    expect(wrapper).toBeDefined();
  });

  it('should have a default state', () => {
    expect(wrapper.state().showingMath6).toEqual(true);
    expect(wrapper.state().showingMath7).toEqual(false);
    expect(wrapper.state().showingMath8).toEqual(false);
  });

  it('should render a filter-section', () => {
    const section = wrapper.find('.filter-section');

    expect(section.length).toEqual(1);
  });

  it('should render a button container', () => {
    const btnContainer = wrapper.find('.grade-btn-container');

    expect(btnContainer.length).toEqual(1);
  });

  it('should render 3 grade buttons', () => {
    const gradeBtn = wrapper.find('.grade-btn');

    expect(gradeBtn.length).toEqual(3);
  });

  it('should render 1 selected grade button', () => {
    const selected = wrapper.find('.grade-selected');

    expect(selected.length).toEqual(1);
  });

  it('should render an options container', () => {
    const optionsContainer = wrapper.find('.options-container');

    expect(optionsContainer.length).toEqual(1);
  });

  it('should render 3 options', () => {
    const options = wrapper.find('.options');

    expect(options.length).toEqual(3);
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
