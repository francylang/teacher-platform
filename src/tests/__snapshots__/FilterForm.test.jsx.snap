// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FilterForm /> should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <FilterForm />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <article
          className="grade-btn-container"
        >
          <button
            className="grade-btn grade-selected"
            onClick={[Function]}
          >
            Grade 6
          </button>
          <button
            className="grade-btn"
            onClick={[Function]}
          >
            Grade 7
          </button>
          <button
            className="grade-btn"
            onClick={[Function]}
          >
            Grade 8
          </button>
        </article>,
        <section
          className="options-container"
        >
          <article
            className="options"
          >
            <FilterSection
              discussions={undefined}
              domain={
                Array [
                  "Ratios & Proportions",
                  "6.RP.A.1",
                  "6.RP.A.2",
                  "6.RP.A.3",
                  "6.RP.A.3.A",
                  "6.RP.A.3.B",
                  "6.RP.A.3.C",
                ]
              }
              renderDiscussions={undefined}
              renderFilteredDiscussions={undefined}
            />
            <FilterSection
              discussions={undefined}
              domain={
                Array [
                  "The Number System",
                  "6.NS.A.1",
                  "6.NS.B.2",
                  "6.NS.B.3",
                  "6.NS.B.4",
                  "6.NS.C.5",
                  "6.NS.C.6",
                  "6.NS.C.7",
                  "6.NS.C.8",
                ]
              }
              renderDiscussions={undefined}
              renderFilteredDiscussions={undefined}
            />
            <FilterSection
              discussions={undefined}
              domain={
                Array [
                  "Expressions & Equations",
                  "6.EE.A.1",
                  "6.EE.A.2",
                  "6.EE.A.3",
                  "6.EE.A.4",
                  "6.EE.B.5",
                  "6.EE.B.6",
                  "6.EE.B.7",
                  "6.EE.B.8",
                  "6.EE.C.9",
                ]
              }
              renderDiscussions={undefined}
              renderFilteredDiscussions={undefined}
            />
            <FilterSection
              discussions={undefined}
              domain={
                Array [
                  "Geometry & Measurement",
                  "6.G.A.1",
                  "6.G.A.2",
                  "6.G.A.3",
                  "6.G.A.4",
                ]
              }
              renderDiscussions={undefined}
              renderFilteredDiscussions={undefined}
            />
            <FilterSection
              discussions={undefined}
              domain={
                Array [
                  "Statistics & Probability",
                  "6.SP.A.1",
                  "6.SP.A.2",
                  "6.SP.A.3",
                  "6.SP.A.4",
                  "6.SP.B.5",
                ]
              }
              renderDiscussions={undefined}
              renderFilteredDiscussions={undefined}
            />
          </article>
          <article
            className="options"
          />
          <article
            className="options"
          />
        </section>,
      ],
      "className": "filter-section",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <button
              className="grade-btn grade-selected"
              onClick={[Function]}
            >
              Grade 6
            </button>,
            <button
              className="grade-btn"
              onClick={[Function]}
            >
              Grade 7
            </button>,
            <button
              className="grade-btn"
              onClick={[Function]}
            >
              Grade 8
            </button>,
          ],
          "className": "grade-btn-container",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Grade 6",
              "className": "grade-btn grade-selected",
              "onClick": [Function],
            },
            "ref": null,
            "rendered": "Grade 6",
            "type": "button",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Grade 7",
              "className": "grade-btn",
              "onClick": [Function],
            },
            "ref": null,
            "rendered": "Grade 7",
            "type": "button",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Grade 8",
              "className": "grade-btn",
              "onClick": [Function],
            },
            "ref": null,
            "rendered": "Grade 8",
            "type": "button",
          },
        ],
        "type": "article",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <article
              className="options"
            >
              <FilterSection
                discussions={undefined}
                domain={
                  Array [
                    "Ratios & Proportions",
                    "6.RP.A.1",
                    "6.RP.A.2",
                    "6.RP.A.3",
                    "6.RP.A.3.A",
                    "6.RP.A.3.B",
                    "6.RP.A.3.C",
                  ]
                }
                renderDiscussions={undefined}
                renderFilteredDiscussions={undefined}
              />
              <FilterSection
                discussions={undefined}
                domain={
                  Array [
                    "The Number System",
                    "6.NS.A.1",
                    "6.NS.B.2",
                    "6.NS.B.3",
                    "6.NS.B.4",
                    "6.NS.C.5",
                    "6.NS.C.6",
                    "6.NS.C.7",
                    "6.NS.C.8",
                  ]
                }
                renderDiscussions={undefined}
                renderFilteredDiscussions={undefined}
              />
              <FilterSection
                discussions={undefined}
                domain={
                  Array [
                    "Expressions & Equations",
                    "6.EE.A.1",
                    "6.EE.A.2",
                    "6.EE.A.3",
                    "6.EE.A.4",
                    "6.EE.B.5",
                    "6.EE.B.6",
                    "6.EE.B.7",
                    "6.EE.B.8",
                    "6.EE.C.9",
                  ]
                }
                renderDiscussions={undefined}
                renderFilteredDiscussions={undefined}
              />
              <FilterSection
                discussions={undefined}
                domain={
                  Array [
                    "Geometry & Measurement",
                    "6.G.A.1",
                    "6.G.A.2",
                    "6.G.A.3",
                    "6.G.A.4",
                  ]
                }
                renderDiscussions={undefined}
                renderFilteredDiscussions={undefined}
              />
              <FilterSection
                discussions={undefined}
                domain={
                  Array [
                    "Statistics & Probability",
                    "6.SP.A.1",
                    "6.SP.A.2",
                    "6.SP.A.3",
                    "6.SP.A.4",
                    "6.SP.B.5",
                  ]
                }
                renderDiscussions={undefined}
                renderFilteredDiscussions={undefined}
              />
            </article>,
            <article
              className="options"
            />,
            <article
              className="options"
            />,
          ],
          "className": "options-container",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <FilterSection
                  discussions={undefined}
                  domain={
                    Array [
                      "Ratios & Proportions",
                      "6.RP.A.1",
                      "6.RP.A.2",
                      "6.RP.A.3",
                      "6.RP.A.3.A",
                      "6.RP.A.3.B",
                      "6.RP.A.3.C",
                    ]
                  }
                  renderDiscussions={undefined}
                  renderFilteredDiscussions={undefined}
                />,
                <FilterSection
                  discussions={undefined}
                  domain={
                    Array [
                      "The Number System",
                      "6.NS.A.1",
                      "6.NS.B.2",
                      "6.NS.B.3",
                      "6.NS.B.4",
                      "6.NS.C.5",
                      "6.NS.C.6",
                      "6.NS.C.7",
                      "6.NS.C.8",
                    ]
                  }
                  renderDiscussions={undefined}
                  renderFilteredDiscussions={undefined}
                />,
                <FilterSection
                  discussions={undefined}
                  domain={
                    Array [
                      "Expressions & Equations",
                      "6.EE.A.1",
                      "6.EE.A.2",
                      "6.EE.A.3",
                      "6.EE.A.4",
                      "6.EE.B.5",
                      "6.EE.B.6",
                      "6.EE.B.7",
                      "6.EE.B.8",
                      "6.EE.C.9",
                    ]
                  }
                  renderDiscussions={undefined}
                  renderFilteredDiscussions={undefined}
                />,
                <FilterSection
                  discussions={undefined}
                  domain={
                    Array [
                      "Geometry & Measurement",
                      "6.G.A.1",
                      "6.G.A.2",
                      "6.G.A.3",
                      "6.G.A.4",
                    ]
                  }
                  renderDiscussions={undefined}
                  renderFilteredDiscussions={undefined}
                />,
                <FilterSection
                  discussions={undefined}
                  domain={
                    Array [
                      "Statistics & Probability",
                      "6.SP.A.1",
                      "6.SP.A.2",
                      "6.SP.A.3",
                      "6.SP.A.4",
                      "6.SP.B.5",
                    ]
                  }
                  renderDiscussions={undefined}
                  renderFilteredDiscussions={undefined}
                />,
              ],
              "className": "options",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": "Ratios & Proportions",
                "nodeType": "function",
                "props": Object {
                  "discussions": undefined,
                  "domain": Array [
                    "Ratios & Proportions",
                    "6.RP.A.1",
                    "6.RP.A.2",
                    "6.RP.A.3",
                    "6.RP.A.3.A",
                    "6.RP.A.3.B",
                    "6.RP.A.3.C",
                  ],
                  "renderDiscussions": undefined,
                  "renderFilteredDiscussions": undefined,
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": "6.NS.A.1",
                "nodeType": "function",
                "props": Object {
                  "discussions": undefined,
                  "domain": Array [
                    "The Number System",
                    "6.NS.A.1",
                    "6.NS.B.2",
                    "6.NS.B.3",
                    "6.NS.B.4",
                    "6.NS.C.5",
                    "6.NS.C.6",
                    "6.NS.C.7",
                    "6.NS.C.8",
                  ],
                  "renderDiscussions": undefined,
                  "renderFilteredDiscussions": undefined,
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": "6.EE.A.2",
                "nodeType": "function",
                "props": Object {
                  "discussions": undefined,
                  "domain": Array [
                    "Expressions & Equations",
                    "6.EE.A.1",
                    "6.EE.A.2",
                    "6.EE.A.3",
                    "6.EE.A.4",
                    "6.EE.B.5",
                    "6.EE.B.6",
                    "6.EE.B.7",
                    "6.EE.B.8",
                    "6.EE.C.9",
                  ],
                  "renderDiscussions": undefined,
                  "renderFilteredDiscussions": undefined,
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": "6.G.A.3",
                "nodeType": "function",
                "props": Object {
                  "discussions": undefined,
                  "domain": Array [
                    "Geometry & Measurement",
                    "6.G.A.1",
                    "6.G.A.2",
                    "6.G.A.3",
                    "6.G.A.4",
                  ],
                  "renderDiscussions": undefined,
                  "renderFilteredDiscussions": undefined,
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": "6.SP.A.4",
                "nodeType": "function",
                "props": Object {
                  "discussions": undefined,
                  "domain": Array [
                    "Statistics & Probability",
                    "6.SP.A.1",
                    "6.SP.A.2",
                    "6.SP.A.3",
                    "6.SP.A.4",
                    "6.SP.B.5",
                  ],
                  "renderDiscussions": undefined,
                  "renderFilteredDiscussions": undefined,
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": "article",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": undefined,
              "className": "options",
            },
            "ref": null,
            "rendered": null,
            "type": "article",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": undefined,
              "className": "options",
            },
            "ref": null,
            "rendered": null,
            "type": "article",
          },
        ],
        "type": "section",
      },
    ],
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <article
            className="grade-btn-container"
          >
            <button
              className="grade-btn grade-selected"
              onClick={[Function]}
            >
              Grade 6
            </button>
            <button
              className="grade-btn"
              onClick={[Function]}
            >
              Grade 7
            </button>
            <button
              className="grade-btn"
              onClick={[Function]}
            >
              Grade 8
            </button>
          </article>,
          <section
            className="options-container"
          >
            <article
              className="options"
            >
              <FilterSection
                discussions={undefined}
                domain={
                  Array [
                    "Ratios & Proportions",
                    "6.RP.A.1",
                    "6.RP.A.2",
                    "6.RP.A.3",
                    "6.RP.A.3.A",
                    "6.RP.A.3.B",
                    "6.RP.A.3.C",
                  ]
                }
                renderDiscussions={undefined}
                renderFilteredDiscussions={undefined}
              />
              <FilterSection
                discussions={undefined}
                domain={
                  Array [
                    "The Number System",
                    "6.NS.A.1",
                    "6.NS.B.2",
                    "6.NS.B.3",
                    "6.NS.B.4",
                    "6.NS.C.5",
                    "6.NS.C.6",
                    "6.NS.C.7",
                    "6.NS.C.8",
                  ]
                }
                renderDiscussions={undefined}
                renderFilteredDiscussions={undefined}
              />
              <FilterSection
                discussions={undefined}
                domain={
                  Array [
                    "Expressions & Equations",
                    "6.EE.A.1",
                    "6.EE.A.2",
                    "6.EE.A.3",
                    "6.EE.A.4",
                    "6.EE.B.5",
                    "6.EE.B.6",
                    "6.EE.B.7",
                    "6.EE.B.8",
                    "6.EE.C.9",
                  ]
                }
                renderDiscussions={undefined}
                renderFilteredDiscussions={undefined}
              />
              <FilterSection
                discussions={undefined}
                domain={
                  Array [
                    "Geometry & Measurement",
                    "6.G.A.1",
                    "6.G.A.2",
                    "6.G.A.3",
                    "6.G.A.4",
                  ]
                }
                renderDiscussions={undefined}
                renderFilteredDiscussions={undefined}
              />
              <FilterSection
                discussions={undefined}
                domain={
                  Array [
                    "Statistics & Probability",
                    "6.SP.A.1",
                    "6.SP.A.2",
                    "6.SP.A.3",
                    "6.SP.A.4",
                    "6.SP.B.5",
                  ]
                }
                renderDiscussions={undefined}
                renderFilteredDiscussions={undefined}
              />
            </article>
            <article
              className="options"
            />
            <article
              className="options"
            />
          </section>,
        ],
        "className": "filter-section",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <button
                className="grade-btn grade-selected"
                onClick={[Function]}
              >
                Grade 6
              </button>,
              <button
                className="grade-btn"
                onClick={[Function]}
              >
                Grade 7
              </button>,
              <button
                className="grade-btn"
                onClick={[Function]}
              >
                Grade 8
              </button>,
            ],
            "className": "grade-btn-container",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Grade 6",
                "className": "grade-btn grade-selected",
                "onClick": [Function],
              },
              "ref": null,
              "rendered": "Grade 6",
              "type": "button",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Grade 7",
                "className": "grade-btn",
                "onClick": [Function],
              },
              "ref": null,
              "rendered": "Grade 7",
              "type": "button",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Grade 8",
                "className": "grade-btn",
                "onClick": [Function],
              },
              "ref": null,
              "rendered": "Grade 8",
              "type": "button",
            },
          ],
          "type": "article",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <article
                className="options"
              >
                <FilterSection
                  discussions={undefined}
                  domain={
                    Array [
                      "Ratios & Proportions",
                      "6.RP.A.1",
                      "6.RP.A.2",
                      "6.RP.A.3",
                      "6.RP.A.3.A",
                      "6.RP.A.3.B",
                      "6.RP.A.3.C",
                    ]
                  }
                  renderDiscussions={undefined}
                  renderFilteredDiscussions={undefined}
                />
                <FilterSection
                  discussions={undefined}
                  domain={
                    Array [
                      "The Number System",
                      "6.NS.A.1",
                      "6.NS.B.2",
                      "6.NS.B.3",
                      "6.NS.B.4",
                      "6.NS.C.5",
                      "6.NS.C.6",
                      "6.NS.C.7",
                      "6.NS.C.8",
                    ]
                  }
                  renderDiscussions={undefined}
                  renderFilteredDiscussions={undefined}
                />
                <FilterSection
                  discussions={undefined}
                  domain={
                    Array [
                      "Expressions & Equations",
                      "6.EE.A.1",
                      "6.EE.A.2",
                      "6.EE.A.3",
                      "6.EE.A.4",
                      "6.EE.B.5",
                      "6.EE.B.6",
                      "6.EE.B.7",
                      "6.EE.B.8",
                      "6.EE.C.9",
                    ]
                  }
                  renderDiscussions={undefined}
                  renderFilteredDiscussions={undefined}
                />
                <FilterSection
                  discussions={undefined}
                  domain={
                    Array [
                      "Geometry & Measurement",
                      "6.G.A.1",
                      "6.G.A.2",
                      "6.G.A.3",
                      "6.G.A.4",
                    ]
                  }
                  renderDiscussions={undefined}
                  renderFilteredDiscussions={undefined}
                />
                <FilterSection
                  discussions={undefined}
                  domain={
                    Array [
                      "Statistics & Probability",
                      "6.SP.A.1",
                      "6.SP.A.2",
                      "6.SP.A.3",
                      "6.SP.A.4",
                      "6.SP.B.5",
                    ]
                  }
                  renderDiscussions={undefined}
                  renderFilteredDiscussions={undefined}
                />
              </article>,
              <article
                className="options"
              />,
              <article
                className="options"
              />,
            ],
            "className": "options-container",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <FilterSection
                    discussions={undefined}
                    domain={
                      Array [
                        "Ratios & Proportions",
                        "6.RP.A.1",
                        "6.RP.A.2",
                        "6.RP.A.3",
                        "6.RP.A.3.A",
                        "6.RP.A.3.B",
                        "6.RP.A.3.C",
                      ]
                    }
                    renderDiscussions={undefined}
                    renderFilteredDiscussions={undefined}
                  />,
                  <FilterSection
                    discussions={undefined}
                    domain={
                      Array [
                        "The Number System",
                        "6.NS.A.1",
                        "6.NS.B.2",
                        "6.NS.B.3",
                        "6.NS.B.4",
                        "6.NS.C.5",
                        "6.NS.C.6",
                        "6.NS.C.7",
                        "6.NS.C.8",
                      ]
                    }
                    renderDiscussions={undefined}
                    renderFilteredDiscussions={undefined}
                  />,
                  <FilterSection
                    discussions={undefined}
                    domain={
                      Array [
                        "Expressions & Equations",
                        "6.EE.A.1",
                        "6.EE.A.2",
                        "6.EE.A.3",
                        "6.EE.A.4",
                        "6.EE.B.5",
                        "6.EE.B.6",
                        "6.EE.B.7",
                        "6.EE.B.8",
                        "6.EE.C.9",
                      ]
                    }
                    renderDiscussions={undefined}
                    renderFilteredDiscussions={undefined}
                  />,
                  <FilterSection
                    discussions={undefined}
                    domain={
                      Array [
                        "Geometry & Measurement",
                        "6.G.A.1",
                        "6.G.A.2",
                        "6.G.A.3",
                        "6.G.A.4",
                      ]
                    }
                    renderDiscussions={undefined}
                    renderFilteredDiscussions={undefined}
                  />,
                  <FilterSection
                    discussions={undefined}
                    domain={
                      Array [
                        "Statistics & Probability",
                        "6.SP.A.1",
                        "6.SP.A.2",
                        "6.SP.A.3",
                        "6.SP.A.4",
                        "6.SP.B.5",
                      ]
                    }
                    renderDiscussions={undefined}
                    renderFilteredDiscussions={undefined}
                  />,
                ],
                "className": "options",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": "Ratios & Proportions",
                  "nodeType": "function",
                  "props": Object {
                    "discussions": undefined,
                    "domain": Array [
                      "Ratios & Proportions",
                      "6.RP.A.1",
                      "6.RP.A.2",
                      "6.RP.A.3",
                      "6.RP.A.3.A",
                      "6.RP.A.3.B",
                      "6.RP.A.3.C",
                    ],
                    "renderDiscussions": undefined,
                    "renderFilteredDiscussions": undefined,
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": "6.NS.A.1",
                  "nodeType": "function",
                  "props": Object {
                    "discussions": undefined,
                    "domain": Array [
                      "The Number System",
                      "6.NS.A.1",
                      "6.NS.B.2",
                      "6.NS.B.3",
                      "6.NS.B.4",
                      "6.NS.C.5",
                      "6.NS.C.6",
                      "6.NS.C.7",
                      "6.NS.C.8",
                    ],
                    "renderDiscussions": undefined,
                    "renderFilteredDiscussions": undefined,
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": "6.EE.A.2",
                  "nodeType": "function",
                  "props": Object {
                    "discussions": undefined,
                    "domain": Array [
                      "Expressions & Equations",
                      "6.EE.A.1",
                      "6.EE.A.2",
                      "6.EE.A.3",
                      "6.EE.A.4",
                      "6.EE.B.5",
                      "6.EE.B.6",
                      "6.EE.B.7",
                      "6.EE.B.8",
                      "6.EE.C.9",
                    ],
                    "renderDiscussions": undefined,
                    "renderFilteredDiscussions": undefined,
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": "6.G.A.3",
                  "nodeType": "function",
                  "props": Object {
                    "discussions": undefined,
                    "domain": Array [
                      "Geometry & Measurement",
                      "6.G.A.1",
                      "6.G.A.2",
                      "6.G.A.3",
                      "6.G.A.4",
                    ],
                    "renderDiscussions": undefined,
                    "renderFilteredDiscussions": undefined,
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": "6.SP.A.4",
                  "nodeType": "function",
                  "props": Object {
                    "discussions": undefined,
                    "domain": Array [
                      "Statistics & Probability",
                      "6.SP.A.1",
                      "6.SP.A.2",
                      "6.SP.A.3",
                      "6.SP.A.4",
                      "6.SP.B.5",
                    ],
                    "renderDiscussions": undefined,
                    "renderFilteredDiscussions": undefined,
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
              ],
              "type": "article",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": undefined,
                "className": "options",
              },
              "ref": null,
              "rendered": null,
              "type": "article",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": undefined,
                "className": "options",
              },
              "ref": null,
              "rendered": null,
              "type": "article",
            },
          ],
          "type": "section",
        },
      ],
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
